package.json

```
@@ -18,50 +18,70 @@
   },
   "repository": "https://github.com/ember-truncate/ember-truncate",
   "scripts": {
-    "build": "ember build",
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "ember test",
-    "test:all": "ember try:each"
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
     "ember-cli-babel": "^7.26.6",
-    "ember-cli-htmlbars": "^3.1.0",
+    "ember-cli-htmlbars": "^5.7.1",
     "ember-diff-attrs": "^0.2.3",
     "ember-lifeline": "^3.1.1",
     "ember-singularity-mixins": "^2.2.0",
     "ember-wormhole": "^0.6.0"
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
+    "@ember/test-helpers": "^2.2.5",
+    "@embroider/test-setup": "^0.37.0",
+    "@glimmer/component": "^1.0.4",
+    "@glimmer/tracking": "^1.0.4",
+    "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
     "ember-ajax": "^5.0.0",
+    "ember-auto-import": "^1.11.2",
     "ember-cli": "~3.26.1",
     "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-eslint": "^5.1.0",
-    "ember-cli-htmlbars-inline-precompile": "^3.0.2",
+    "ember-cli-deprecation-workflow": "^1.0.1",
     "ember-cli-inject-live-reload": "^2.0.2",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-template-lint": "^2.0.2",
-    "ember-cli-uglify": "^3.0.0",
+    "ember-cli-terser": "^4.0.1",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.1",
     "ember-faker": "^1.6.0",
     "ember-load-initializers": "^2.1.2",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-qunit": "^4.6.0",
+    "ember-page-title": "^6.2.1",
+    "ember-qunit": "^5.1.4",
     "ember-resolver": "^8.0.2",
     "ember-source": "~3.27.2",
     "ember-source-channel-url": "^3.0.0",
+    "ember-template-lint": "^3.2.0",
     "ember-try": "^1.4.0",
-    "eslint-plugin-ember": "^8.14.0",
+    "eslint": "^7.23.0",
+    "eslint-config-prettier": "^8.1.0",
+    "eslint-plugin-ember": "^10.3.0",
     "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.3.1",
     "loader.js": "^4.7.0",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^2.2.1",
+    "qunit": "^2.14.1",
     "qunit-dom": "^1.6.0"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "10.* || >= 12"
```

Node 10 is EOL. If we're going to make this part of a major, let's just remove this.

```suggestion
    "node": ">= 12"
```

---

https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/package.json#L68

This isn't the only place this change would have to be made, since the Travis config uses Node 10. That said, you probably want to move from Travis to GitHub Actions in a separate PR anyway.

---

Correct. We'll want to update in our CI configs.